requirements-lint.txt

```diff
@@ -1,3 +1,9 @@
+# pycodestyle==2.4.0
+# To avoid [this](https://github.com/PyCQA/pycodestyle/issues/635) bug
+# we need pycodestyle 2.4.0 but unfortunately this does not work with flake8
+# due to https://github.com/PyCQA/pycodestyle/issues/741
+# Once a new version of flake8 is released we can specify the latest pycodestyle
+# version here
```

Comment:
Should we add a follow-up issue for this?

Reply:
Naah the minute flake8 is released this is solved and we can update it and remove this comment